staging/src/k8s.io/client-go/examples/leader-election/main.go

```diff
@@ -57,6 +58,7 @@ func main() {
 	var leaseLockName string
 	var leaseLockNamespace string
 	var id string
+	var startedLeading int32 // Use int32 for atomic operations
```

Comment:
if you just want to use it as a bool use the specific type, golang has added `atomic.Bool` recently

Reply:
A bit awkward, I didn't notice this. Thank you for the reminder. :joy: 
fixed